src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -105,18 +104,18 @@ const removeNoteAction = (ctx, note) => (onMouseDown) => {
 const moveNoteAction = (ctx, note) => (onMouseDown, onMouseMove) => {
   let startPosition
   let startNote = _.clone(note)
-  onMouseDown(e => { startPosition = getLocal(e) })
-  onMouseMove(e => { onNoteMouseMove(ctx, e, startPosition, startNote) })
+  onMouseDown(local => { startPosition = local })
+  onMouseMove(local => { onNoteMouseMove(ctx, local, startPosition, startNote) })
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use 'esversion
*  Too many errors. (58% scanned).


<!-- Comment by [SideCI](https://sideci.com) -->